metacat-functional-tests/src/functionalTest/groovy/com/netflix/metacat/MetacatSmokeSpec.groovy

```diff
@@ -261,6 +262,48 @@ class MetacatSmokeSpec extends Specification {
         'invalid-catalog'               | 'smoke_db1'  | 'z'                 | true   | false   | MetacatNotFoundException.class
     }
 
+    @Unroll
+    def "Test create/update iceberg table"() {
+        given:
```

Comment:
Why does the test stays in SmokeSpec instead of the functionalSpec? Feel majority of the functional tests are in the functionalSpec, and some of them are here. I missed it in the first place since I thought we should put the functionalSpec.